### PR TITLE
Add hideAlphaControl property to ClockPicker to optional turn off alp…

### PR DESCRIPTION
**Pull request by timtong1982**
…ha slider

#### Pull request checklist

- [x] Addresses an existing issue: #0000
- [x] Include a change request file if publishing 
- [x] New feature, bugfix, or enhancement
  - [x] Includes tests
- [x] Documentation update

#### Description of changes

(give an overview)

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.Props.ts [hunk 1]
```diff
@@ -8,4 +8,9 @@ export interface IColorPickerProps {
    * Callback issued when the user changes the color.
    */
   onColorChanged?: (color: string) => void;
+
+  /**
+   * The setting of whether hide alpha control slider.
+   */
+  hideAlphaControl?: boolean;
```
Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.Props.ts]: maybe better name: `alphaSliderHidden`: boolean

File: packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.test.tsx [hunk 2]
```diff
@@ -36,4 +37,12 @@ describe('ColorPicker', () => {
     expect(component.state.color.hex).to.equal('aeaeae');
     expect(color).to.equal(newColor);
   });
+
+  it('Hide alpha control slider', () => {
```
Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.test.tsx]: Typo, should be plural 'Hides'

File: packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.test.tsx [hunk 3]
```diff
@@ -36,4 +37,12 @@ describe('ColorPicker', () => {
...
+    let component = ReactTestUtils.renderIntoDocument(
+      <ColorPicker color='#FFFFFF' hideAlphaControl={ true } />
+    ) as ColorPicker;
+
+    expect(component.props.hideAlphaControl).to.equal(true);
```
Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.test.tsx]: I think this should validate that the DOM doesn't render the slider.

File: packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.test.tsx [hunk 4]
```diff
@@ -13,6 +13,7 @@ describe('ColorPicker', () => {
     ) as ColorPicker;
 
     expect(component.state.color.hex).to.equal('ffffff');
+    expect(component.props.hideAlphaControl).to.equal(undefined);
```
Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.test.tsx]: not sure why this is necessary.

File: packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.test.tsx (hunk 2 above)
Comment (timtong1982) [inline packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.test.tsx]: fixed

**Pull request merged**